src/service/onesignal/index.ts

```diff
@@ -31,6 +35,7 @@ const rebuildAndSubscribeTags = async () => {
     });
   } catch (e) {
     console.log('error rebuilding and subscribing tags ', e?.message);
+    Monitoring.logError('error rebuilding and subscribing tags ', e?.message);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good that you are logging errors, but it would be better to log the entire error object instead of just the message. This will provide more context if an error occurs, such as the stack trace.
```diff
-    Monitoring.logError('error rebuilding and subscribing tags ', e?.message);
+    Monitoring.logError('error rebuilding and subscribing tags ', e);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
done on next commit